README.Rmd

```diff
@@ -967,10 +976,61 @@ above under [Computing BMI percentiles and Z-scores](#bmi).
 same `syngrowth` synthetic example dataset as `growthcleanr`, with results
 included.
 
+## <a name="nhanes"></a>NHANES reference medians
+
+`growthcleanr` releases up to 1.2.4 offered two options for recentering medians, either
```

Comment:
Wondering if it might be nice here to put a link to releases/tags?

Reply:
Good idea! Added a link under Changes section below.